### PR TITLE
Add new company - Guidewire

### PR DESCRIPTION
**Pull request by dstronczak**
Adds the following events or companies:
 - Guidewire (Company)
 - 

### Checklist

 - [ x ] I have updated the event or company counts
 - [ x ] I have put the most recent relevant date in the "Last Update" column
 - [ ] I have linked to the article about the change, not the company's website

### CODE REVIEW

**Comment by dstronczak** [comment]
Done.

File: README.md [hunk 1]
```diff
@@ -7,6 +7,173 @@ This is the running list of what in tech has been affected by COVID-19. Pull req
...
+| Wikimedia Foundation[[1]](https://wikimediafoundation.org/news/2020/03/06/wikimedia-foundation-will-close-san-francisco-office-and-encourage-remote-work-for-march-2020-amidst-covid-19-concerns/) | Encouraged, required for San Francisco office | Restricted | Restricted | Restricted | 2020-03-06 |
+| Wish | Encouraged, required for some locations | Restricted | Restricted | Restricted | 2020-03-05 |
+| Workday | Required | Restricted | Restricted | Restricted | 2020-03-09 |
+| Yelp | Encouraged | Restricted | ? | ? | 2020-03-03 |
+| Zappos | Required | Restricted | Restricted | Restricted | 2020-03-11 |
+| Zenefits | Required | Restricted | ? | ? | 2020-03-10 |
+| Zeus Living | Required | Restricted | Restricted | Restricted | 2020-03-10 |
+| Zillow | Encouraged | Restricted | ? | ? | 2020-03-06 |
+| Zulily | Encouraged | Restricted | Restricted | ? | 2020-03-07 |
+=======
```
Comment (drewbrew) [inline README.md]: Please drop all your changes here as well.

File: README.md [hunk 2]
```diff
@@ -7,6 +7,173 @@ This is the running list of what in tech has been affected by COVID-19. Pull req
...
+| GetYourGuide[[1]](https://twitter.com/JohannesReck/status/1237361981008592896) | Required | Restricted | Restricted | Restricted | 2020-03-11 |
+| GitHub[[1]](https://github.blog/2020-03-03-covid-19-update-supporting-our-employees-and-community/) | Encouraged | Restricted | ? | ? | 2020-03-04 |
+| Glassdoor[[1]](https://www.glassdoor.com/blog/glassdoor-coronavirus/) | Required | Restricted | Restricted | ? | 2020-03-05 |
+| Glitch | Required | Restricted | Restricted | Restricted | 2020-03-09 |
+| Google[[1]](https://www.theguardian.com/world/2020/mar/04/coronavirus-google-tech-dublin-twitter-work-from-home#maincontent) [[2]](https://www.reuters.com/article/us-health-coronavirus-alphabet/google-recommends-all-north-america-employees-to-work-from-home-idUSKBN20X38J?feedType=RSS&feedName=technologyNews) | Encouraged / Geo-specific | Restricted | Restricted | ? | 2020-03-11 |
+| Granular.ag | Required for SF / All others allowed | Restricted | Restricted for SF | Restricted | 2020-03-05 |
+| Groupon | Encouraged | Restricted | ? | Restricted | 2020-03-06 |
+| Guidewire | Encouraged | Restricted | Restricted | Restricted | 2020-03-11 |
```
Comment (dstronczak) [inline README.md]:
```suggestion
| Guidewire | Encouraged | Restricted | Restricted | Restricted | 2020-03-12 |
```

Comment (drewbrew) [reply README.md]: Sorry, I meant *remove* all the companies that your PR is adding. GitHub auto-generates the list from the YAML, so if I merge this, we'll have duplicate companies.

In other words, your PR should have zero changes to README.md.

Comment (dstronczak) [reply README.md]: Oh, I see. Done.

**Comment by drewbrew** [comment]
I meant removing literally all the companies, as in https://github.com/phildini/stayinghomeclub/pull/436/commits/42a2e2b6c065b0b1a00d145b62b07d50d1ba336c. 

Thanks for submitting this!

**Pull request merged**